Druid/DRControlFlowGraph.class.st

```diff
@@ -551,7 +555,9 @@ DRControlFlowGraph >> initialize [
 
 	nextBasicBlockId := 0.
 	initialBasicBlock := self privateNewBasicBlock.
-	numArgs := 0
+	numArgs := 0.
+
+	markedBlocks := Set new.
```

Comment:
Why this is a collection in the CFG and not a marker in the basic block?

Reply:
I put it here just because it's only used in `DRControlFlowGraph>>inspectionCFG`, but as you prefer